controllers/movies.js

```diff
@@ -0,0 +1,70 @@
+var Movie = require('../models/movie')
+var Rental = require('../models/rental')
+
+MoviesController = {
+  locals: {
+    title: 'MOVIES MOVIES MOVIES'
+  },
+
+  getMovies: function(req, res) {
+    Movie.all (function (error, movies) {
+      if (error) {
+        var err = new Error("Error retrieving movie list:\n" + error.message);
+        err.status = 500;
+        next(err);
+      } else {
+        res.json(movies)
+      }
+    })
+  },
+
+  getMoviesShow: function(req, res, next) {
+    Movie.find(req.params.title, function(error, movie) {
```

Comment:
I've never used this framework before but it's cool how much this looks like rails